core/tests/regression_tests.rs

```diff
@@ -414,18 +415,19 @@ swf_tests! {
 // Eventually we can hopefully make some of these match exactly (see #193).
 // Some will probably always need to be approx. (if they rely on trig functions, etc.)
 swf_tests_approx! {
-    (local_to_global, "avm1/local_to_global", 1, 0.051),
-    (stage_object_properties, "avm1/stage_object_properties", 5, 0.051),
-    (stage_object_properties_swf6, "avm1/stage_object_properties_swf6", 4, 0.051),
-    (movieclip_getbounds, "avm1/movieclip_getbounds", 1, 0.051),
-    (edittext_letter_spacing, "avm1/edittext_letter_spacing", 1, 15.0), // TODO: Discrepancy in wrapping in letterSpacing = 0.1 test.
-    (edittext_align, "avm1/edittext_align", 1, 3.0),
-    (edittext_margins, "avm1/edittext_margins", 1, 5.0), // TODO: Discrepancy in wrapping.
-    (edittext_tab_stops, "avm1/edittext_tab_stops", 1, 5.0),
-    (edittext_bullet, "avm1/edittext_bullet", 1, 3.0),
-    (edittext_underline, "avm1/edittext_underline", 1, 4.0),
-    (as3_coerce_string_precision, "avm2/coerce_string_precision", 1, 10_000_000.0),
-    (as3_divide, "avm2/divide", 1, 0.0), // TODO: Discrepancy in float formatting.
+    (local_to_global, "avm1/local_to_global", 1, 0.051, None),
+    (stage_object_properties, "avm1/stage_object_properties", 5, 0.051, None),
+    (stage_object_properties_swf6, "avm1/stage_object_properties_swf6", 4, 0.051, None),
+    (movieclip_getbounds, "avm1/movieclip_getbounds", 1, 0.051, None),
+    (edittext_letter_spacing, "avm1/edittext_letter_spacing", 1, 15.0, None), // TODO: Discrepancy in wrapping in letterSpacing = 0.1 test.
+    (edittext_align, "avm1/edittext_align", 1, 3.0, None),
+    (edittext_margins, "avm1/edittext_margins", 1, 5.0, None), // TODO: Discrepancy in wrapping.
+    (edittext_tab_stops, "avm1/edittext_tab_stops", 1, 5.0, None),
+    (edittext_bullet, "avm1/edittext_bullet", 1, 3.0, None),
+    (edittext_underline, "avm1/edittext_underline", 1, 4.0, None),
+    (as3_coerce_string_precision, "avm2/coerce_string_precision", 1, 10_000_000.0, None),
```

Comment:
Would the relative epsilon feature be more useful with this test? I had to set it's absolute epsilon to something comically large.

Reply:
(In case it isn't obvious I'm specifically referring to `as3_coerce_string_precision`)